### ISSUE TITLE
M365 Bootcamp for makers

### ISSUE THREAD

**Issue by LuiseFreese** [MicrosoftDocs/microsoft-365-community#354]
#### Article suggestion

I attended Global M365 Dev bootcamp - now blogging about this experience (it was awesome and totally doable, even if you are not very dev-y) to make some noise about it, as recording of the labs is available as well. As this could get more people into contributing and using M365 and Graph Toolkit /SharePoint/ Teams... would you think that it would fit into community docs? Or is this only for How-to articles?

Draft link here - still needs spell check, grammar check, pics and links etc. 
[m365Bootcamp for makers](https://1drv.ms/u/s!AsD6nReCikUeg6pHzRTeNZha7Zguiw?e=FaIENr)

**Comment by sympmarc** [comment]
Sorry for the delayed reply! This is great stuff, but I think it's a little to specific for this repo. If you'd be interested in abstracting it somehow to it remains timely (even in there are no longer bootcamps, for example, or the labs are no longer relevant), I think it could be really helpful to people.

It feel like some of your higher-level messages are things people should definitely hear (I also saw some of the Twitter discussions):
* Don't be afraid to dip your toe into coding - or becoming a "maker"
* Learning is good!
* Doing this stuff gives you better connections into the community
* etc,

In other words, discussing why someone might do something like a bootcamp and what generally they can get out of it. Make sense?

**Comment by LuiseFreese** [comment]
It does! Let me change to birds view for that!

**Comment by LuiseFreese** [comment]
will need a couple of days more... carzy times.

**Comment by eemancini** [comment]
Hello @LuiseFreese, just checking in on your thoughts for this article. Let us know if you need any support :) Thanks for being such a frequent contributor!

**Comment by LuiseFreese** [comment]
Hey Emily, since this is now more than a year ago, I published this in 2020 on my personal blog -

https://www.m365princess.com/blogs/2020-10-19-m365-developer-bootcamp/

**Comment by eemancini** [comment]
Thanks so much for the update. Since this is a more technical article, that is a great place for it or even the PnP blog :)
